client/lib/pdf-label-utils/index.js

```diff
@@ -42,8 +42,8 @@ const _getPDFURL = ( paperSize, labels, baseURL, nonce ) => {
 	const params = {
```

Comment:
Worth leaving a comment explaining why it's not just using an array, and warning against using QS arrays in the future?